collector/perflib.go

```diff
@@ -0,0 +1,131 @@
+package collector
+
+import (
+	"fmt"
+	"reflect"
+
+	"github.com/leoluk/perflib_exporter/perflib"
+	"github.com/prometheus/common/log"
+)
+
+func getPerflibSnapshot() (map[string]*perflib.PerfObject, error) {
+	objects, err := perflib.QueryPerformanceData("Global")
+	if err != nil {
+		return nil, err
+	}
+
+	indexed := make(map[string]*perflib.PerfObject)
+	for _, obj := range objects {
+		indexed[obj.Name] = obj
+	}
+	return indexed, nil
+}
+
+const (
+	perfCounterRawcountHex         uint32 = 0
```

Comment:
I would write these numbers in hex as there's less symmetry in their decimal representation